src/main/java/org/example/zad3.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class zad3 {
+
+    static Scanner in = new Scanner(System.in);
+    public static void main(String[] args) {
+
+        System.out.print("Enter weight: ");
+        int w = getInt(50, 500);
```

Comment:
The variable and method are not explaining what they are doing. Be more descriptive.